server/app/routes/api/v1/va/ambassadors.js

```diff
@@ -387,6 +387,10 @@ async function ambassadorPayouts(ambassador, neode) {
     payouts.push(obj);
   }));
 
+  payouts.sort((p1, p2) => {
+    return p1.disbursed_at < p2.disbursed_at ? -1 : 1;
```

Comment:
@brodavi if disbursed_at is undefined on one side of comparison, they will probably give unpredictable result. May be we should check for null/undefined before comparing.